### DISCUSSION TITLE
Contextual menus don't load

### DISCUSSION THREAD

**Discussion by gzunzu** [community/community#46456] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hi.

I'm having many troubles for managing GitHub repositories or my own profile settings through the web. No contextual menu loads, so I cannot access to anything that requires that feature: profile options, managing branches, etc. It's a serious bug; I just could acces to my repostories because I manually typed my username on the browser searchbar.

I can't neither upload attachments to this issue. This doesn't seem to be related to the contextual menu bug.

I'm using Google Chrome. Can you help me, please?

Thanks in advance.

**Comment by chris-c-thomas** [comment]
You're talking about this menu search bar at the top right?

<img width="1166" alt="Screenshot 2023-02-05 at 6 04 03 PM" src="https://user-images.githubusercontent.com/22353987/216854100-3f390d82-93cd-4734-beec-c063d91dc8d3.png">


It tends to depend on what filters you place as well sometimes.

<img width="1307" alt="Screenshot 2023-02-05 at 6 05 19 PM" src="https://user-images.githubusercontent.com/22353987/216854168-f2ed5001-0086-4c1f-9c47-187308106256.png">

Or are you talking about the search bar that's on your repository profile page?

<img width="942" alt="Screenshot 2023-02-05 at 6 07 01 PM" src="https://user-images.githubusercontent.com/22353987/216854254-78d319d0-b882-4ab7-9535-ff021002d77f.png">

**Reply by gzunzu** [reply]
Hi!

Thanks for replying.

I was talking about 2 different menus, but I checked the ones you show in those pics and they also don't work, as well as many other common features. I tried to download an alternative browser (Mozilla Firefox) and everything works well. I've checked settings in Chrome and nothing seems to be wrong, JavaScript is enabled.

I'm writting this comment using Firefox, so I can upload images now. Here there are some examples.

- Trying to get to my profile:
![Error1](https://user-images.githubusercontent.com/78328583/217038485-5265026f-fbfb-49d6-8525-eaa5c05b3fdc.png)
- Trying to rename a branch:
![Error2](https://user-images.githubusercontent.com/78328583/217038489-27f27fd4-8cc5-4862-8b6a-336933074b29.png)
- Trying the general search field (it stays permanently loading):
![Error3](https://user-images.githubusercontent.com/78328583/217038492-f6b6e92b-5a03-4ee7-8f23-680d05a4fd16.png)
-Trying to search into my repositories:
![Error4](https://user-images.githubusercontent.com/78328583/217038495-2f61eeb9-c678-4d8d-93d1-e0e9ad39eabb.PNG)
-Commits information is not showed:
![Error5](https://user-images.githubusercontent.com/78328583/217038498-9ac9ae3f-0a04-4f26-b8b4-eeb361aad485.PNG)

I wish to keep using Chrome and I guess this is something bad on my side; otherwise many users would be experimenting this problems. Do you have any idea about what could be wrong?

Thanks again,

**Comment by chris-c-thomas** [comment, marked as answer]
Yeah I just tested on my Chrome and it's working fine. I suspect it's either a plugin/extension that's blocking this or are you on a VPN? I don't think that last one should matter too much unless the VPN is blocking things too. ProtonVPN for instance can give me trouble getting things to load on the webpage.

Idk which verstion of Chrome you have: stable, beta, dev, and canary but maybe try downloading one you don't have and use a fresh copy. Don't sync your profile or anything so that no settings or extensions get installed.

**Reply by chris-c-thomas** [reply]
Not being able to load your profile photo dropdown makes me think it's an extension.

**Reply by gzunzu** [reply]
You were so right! I tried to deactivate every extension and it worked. Then I start enabling them one by one and the weirdest thing is that the problematic one was... **minesweeper-game**! I'm a little shocked right now, but so happy to be back navigating in GitHub using my usual browser.

Many thanks for your guidance, Chris. I feel lucky you replyed. And forgive me if my explanation wasn't accurate or it was poorly written, I'm not a native english speaker.

**Reply by chris-c-thomas** [reply]
I'm glad we were able to get it working! Maybe see if you can file a bug report with the developer of the game, I'm sure they'd actually appreciate something like that!

**Reply by gzunzu** [reply]
Done! I just sent it. Hope it helps. 🙂
